src/qt/bitcoingui.cpp

```diff
@@ -123,7 +123,11 @@ BitcoinGUI::BitcoinGUI(const PlatformStyle *_platformStyle, const NetworkStyle *
     spinnerFrame(0),
     platformStyle(_platformStyle)
 {
-    GUIUtil::restoreWindowGeometry("nWindow", QSize(850, 550), this);
+    QSettings settings;
+    if (!restoreGeometry(settings.value("MainWindowGeometry").toByteArray())) {
+        // Restore failed (perhaps missing setting), center the window
+        move(QApplication::desktop()->screen()->rect().center() - rect().center());
```

Comment:
I believe this is wrong (don't have 2nd screen to test) since `rect()` is relative to widget as in `QRect(0, 0, width(), height())`.

Reply:
Should be something like?
```cpp
move(QApplication::desktop()->availableGeometry().center() - geometry().center());
```

Reply:
Fixed, thanks :-)